lighthouse-core/test/audits/dobetterweb/uses-http2-test.js

```diff
@@ -61,4 +61,22 @@ describe('Resources are fetched over http/2', () => {
       assert.ok(auditResult.displayValue === '');
     });
   });
+
+  it('results are correct when some requests are handled by service worker', () => {
+    const clonedNetworkRecords = JSON.parse(JSON.stringify(networkRecords));
+    clonedNetworkRecords.forEach(record => {
+      // convert http 1.x to service worker requests
+      if (record.protocol === 'http/1.1') {
+        record.fromServiceWorker = true;
```

Comment:
@wardpeet think you could fix the bug over in network-records-to-devtools-log? it's reading the wrong property name when doing the `networkRecordsToDevtoolsLog` round-trip :)